src/js/dune

```diff
@@ -1,5 +1,8 @@
 (env
  (dev
+  (js_of_ocaml
+   (flags (:standard --enable effects))
+   (build_runtime_flags (:standard --enable effects)))
```

Comment:
You should also set the flags for non `dev` profiles (e.g. release).

Reply:
Thanks!